modules/ROOT/pages/data-bridge.adoc

```diff
@@ -0,0 +1,141 @@
++
+----
+    {
+      "access_token": "5242XXXX-XXXX-XXXX-XXXX-XXXXXXXXfec7",
+      "token_type": "bearer",
+      "redirectUrl": "/home/"
+    }
+----
++
+. Note the `access_token` value.
++
+. Use the next `cURL` example to register an external client in Anypoint Platform for OpenID authentication.
++
+Replace the access token in the authorization header with the `access_token` value.
++
+Replace `[YourOrgDomain]` with your Salesforce organization domain name. Find the domain name by logging in to your Salesforce organization, clicking *Setup*, and clicking the section *My Domain*.
++
+----
+    curl -X POST \
+      https://anypoint.mulesoft.com/accounts/api/connectedApplications \
+      -H 'Content-Type: application/json' \
+      -H 'Accept: */*' \
+      -H 'Accept-Encoding: gzip, deflate' \
+      -H 'Authorization: Bearer 5242XXXX-XXXX-XXXX-XXXX-XXXXXXXXfec7' \
+      -H 'Host:anypoint.mulesoft.com' \
+      -d '{"client_name":"ACM Integration App","grant_types":["authorization_code","refresh_token"],"redirect_uris":["https://[YourOrgDomain].my.salesforce.com/services/authcallback/Anypoint"],"scopes":["full","offline_access"],"public_keys":[]}'
+----
++
+. The `cURL` command returns a response like this:
++
+----
+    {
+      "client_id": "5fafXXXXXXXXXXXXXXXXXXXXXXXX29c9",
+      "client_secret": "9509XXXXXXXXXXXXXXXXXXXXXXXXC10E",
+      "client_name": "ACM Integration App",
+      "redirect_uris": [
+        "https://[YourOrgDomain].my.salesforce.com/services/authcallback/Anypoint"
+      ],
+      "grant_types": [
+        "authorization_code",
+        "refresh_token"
+      ],
+      "public_keys": [],
+      "scopes": [
+        "offline_access",
+        "full"
+      ],
+      "enabled": true,
+      "owner_org_id": "f377XXXX-XXXX-XXXX-XXXX-XXXXXXXX9d08",
+      "as_id": "anypoint"
+    }
+----
++
+. Note the `client_id` and `client_secret` values.
+
+== Create an Authentication Provider in Your Salesforce Organization [[create-auth-provider]]
+
+NOTE: If your Anypoint Platform organization is hosted in the EU, change `+https://anypoint.mulesoft.com+` to `+https://eu1.anypoint.mulesoft.com+`.
+
+. In *Setup*, search for *Auth. Providers* using the *Quick Find* box and click *Auth. Providers* -> *New*.
+. In *Provider type* select *Open ID Connect*.
+. Set *Name* to `Anypoint`.
+. Set *Consumer Key* to the application ID of the connected app created in Anypoint Platform.
++
+This ID was returned by the Anypoint UI at the end of the task <<create-a-connected-app-with-the-user-interface>> or returned by the Anypoint API call at the end of the task <<create-a-connected-app-with-an-api-call>>.
++
+. Set *Consumer Secret* to the application secret of the connected app created in Anypoint Platform.
++
+This secret was returned by the Anypoint UI at the end of the task <<create-a-connected-app-with-the-user-interface>> or returned by the Anypoint API call at the end of the task <<create-a-connected-app-with-an-api-call>>.
++
```

Comment:
ditto previous
If we make any refs. at all, I suggest we ref. FORWARD, at the point that the ID and secret are first received, saying that the user needs them for <this task>, instead of referring BACKWARD to that time here.